core/src/main/java/org/apache/accumulo/core/fate/user/UserFateStore.java

```diff
@@ -231,7 +228,9 @@ public Optional<FateTxStore<T>> tryReserve(FateId fateId) {
     // Create a unique FateReservation for this reservation attempt
     FateReservation reservation = FateReservation.from(lockID, UUID.randomUUID());
 
-    FateMutator.Status status = newMutator(fateId).putReservedTx(reservation).tryMutate();
+    // requiring any status prevents creating an entry if the fate id doesn't exist
+    FateMutator.Status status =
```

Comment:
Nice comment here.  Interesting, I suppose the status check was not needed previously because it was assumed the reservation column always existed if the fate operation existed.

Reply:
Yeah, before this, `putReservedTx` only worked if the reservation column had a NOT_RESERVED value, which was only set on creation and unreservation.